imap/cyr_expire.c

```diff
@@ -210,6 +212,8 @@ static void cyr_expire_init(const char *progname, struct cyr_expire_ctx *ctx)
 
 static void cyr_expire_cleanup(struct cyr_expire_ctx *ctx)
 {
+    if (ctx->args.freeme) free(ctx->args.freeme);
+
```

Comment:
Should we be zeroing out more stuff here?

Reply:
Just checked and yeah it's fine -- `ctx->args.freeme` is the only thing that was malloced here.  And we only call `cyr_expire_cleanup()` right before `exit()`, so we don't need to zero out the structures for later reuse or anything. :)